.github/workflows/updatecli.yaml

```diff
@@ -0,0 +1,33 @@
+name: updatecli
+on:
+  # Allow to be run manually
+  workflow_dispatch:
+  schedule:
+    # Run once per week (to avoid alert fatigue)
+    - cron: '0 2 * * 1' # Every monday at 2am UTC
+jobs:
+  updatecli:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install updatecli CLI
+        env:
+          UPDATECLI_VERSION: "0.33.3"
```

Comment:
is there a reason you aren't using the github action? https://github.com/updatecli/updatecli-action

generally it's better to use that as dependabot / renovate will do it.
normally the reason to not use the action is if you need an extra tool installed that isn't in the container it runs in